rebar.lock

```diff
@@ -17,6 +17,7 @@
       {ref,"60a335668a60328a29f9731b67c4a0e9e3d50ab6"}},
   0},
  {<<"bear">>,{pkg,<<"bear">>,<<"0.8.7">>},2},
+ {<<"bucs">>,{pkg,<<"bucs">>,<<"1.0.16">>},1},
```

Comment:
I see bucrandom process starts though seems ok.

Reply:
If anyone knows of a more lightweight way than `ephemeral` to create temporary files it can be changed!